### ISSUE TITLE
为什么“进程已结束，退出代码为 0”但是没有任何结果出来呢？

### ISSUE THREAD

**Issue by franklee24** [NLPxiaoxu/Easy_Bert_classify#2]
D:\Anaconda\python.exe D:/desk/Easy_Bert_classify-master/Easy_Bert_classify-master/bert_classify.py
WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

WARNING:tensorflow:From D:\Anaconda\lib\site-packages\bert\modeling.py:233: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.Dense instead.
WARNING:tensorflow:From D:\Anaconda\lib\site-packages\tensorflow_core\python\layers\core.py:187: Layer.apply (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
Please use `layer.__call__` method instead.
WARNING:tensorflow:From D:/desk/Easy_Bert_classify-master/Easy_Bert_classify-master/bert_classify.py:30: dropout (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dropout instead.
WARNING:tensorflow:From D:\Anaconda\lib\site-packages\tensorflow_core\python\ops\math_grad.py:1375: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where

进程已结束，退出代码为 0
